src/configuration.zig

```diff
@@ -130,7 +130,11 @@ pub fn configChanged(config: *Config, runtime_zig_version: *?ZigVersionWrapper,
         defer allocator.free(result.stdout);
         defer allocator.free(result.stderr);
 
-        var d = try std.fs.cwd().openDir(builtin_creation_dir.?, .{});
+        var d = if (std.fs.path.isAbsolute(builtin_creation_dir.?))
```

Comment:
I don't believe that this needs to be changed as openDir from cwd should handle absolute paths properly without this check :)